test/views/TopStories/test.index.js

```diff
@@ -0,0 +1,50 @@
+import React from 'react';
+import assert from 'assert';
+import sinon from 'sinon';
+import { shallow } from 'enzyme';
+import TopStories from '../../../views/TopStories/index.jsx';
+
+function selectDate(storiesWrapper) {
+  storiesWrapper.find('.sort-option').nodes[0].props.children[1].props.children.props.onChange({target: { value : 'date'}});
+}
+
+function selectRelevance(storiesWrapper) {
```

Comment:
it would be even better if you didn't have to pass in the wrapper at all and just refer to it within the describe blocks - javascript functions are hoisted so you can just embed them within the describe block and they will have access to your `let wrapper;` that ultimately gets assigned within the `beforeEach`